libs/pages/elewa/contact/src/lib/components/contact-form/contact-form.component.html

```diff
@@ -0,0 +1,85 @@
+<div class="contact-main">
+  <div class="contact-info">
+    <h1 class="heading">Contact</h1>
+    <p>
+      Looking for a quick answer to your question? Check out our FAQs. <br />
+      <br />
+      <br />
+      For everything else, drop us a line and we'll get back to you.
+    </p>
+    <div class="icons">
```

Comment:
Your design is great but still not quite there.
The anchor text for privacy policy is white not blue
The margin between input fields is not consistent
The width of the input fields is not consistent
No mobile design for responsiveness